web/src/components/pages/dashboard/stats/stats-card.tsx

```diff
@@ -31,13 +31,16 @@ export default function StatsCard({
   scrollable?: boolean;
   currentLocale?: string;
 }) {
-  const [locale, setLocale] = useLocalStorageState(DEFAULT_LOCALE_OPTION, id);
+  const [locale, setLocale] = useLocalStorageState(
+    DEFAULT_LOCALE_OPTION,
+    `${id}${currentLocale}`
```

Comment:
We now store a value per stats card/tab before it didn't account for the language tab.